### PR TITLE
Migrate routes to native classes

### PR DESCRIPTION
**Pull request by locks**

### CODE REVIEW

File: app/routes/application.js [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-  },
+  }
 
-  actions: {
-    didTransition() {
-      this.flashMessages.step();
-    },
-  },
-});
+  @action
```
Comment (Turbo87) [inline app/routes/application.js]: I'm a little worried about this because I've had issues with the `@action` decorator on routes in the past 🤔

Comment (locks) [reply app/routes/application.js]: Do you remember what the issue was? I tried checking the literature and this seemed the correct way to upgrade :/

Comment (Turbo87) [reply app/routes/application.js]: nope, unfortunately I don't remember the specifics, but I'd sleep better if I knew that we had better test coverage for these things :-/

**Pull request closed without merging**